src/pool/pool_disjoint.c

```diff
@@ -29,10 +28,30 @@
 
 static char *DEFAULT_NAME = "disjoint";
 
+enum {
```

Comment:
please add a comment how this is used
as I understand you create single bitmask instead of multiple bool flags ..  maybe you could keep this enum close to "params_overriden" filed?